opencl/source/cl_device/cl_device_info.cpp

```diff
@@ -233,6 +233,12 @@ cl_int ClDevice::getDeviceInfo(cl_device_info paramName,
         src = deviceInfo.extensionsWithVersion.data();
         retSize = srcSize = deviceInfo.extensionsWithVersion.size() * sizeof(cl_name_version);
         break;
+    case CL_DEVICE_PCI_BUS_INFO_INTEL:
+        if (isPciBusInfoValid()) {
+            src = &deviceInfo.pciBusInfo;
+            retSize = srcSize = sizeof(deviceInfo.pciBusInfo);
+        }
+        break;
```

Comment:
Do we have API ULT  for this switch?

Reply:
The test has been added. Please see new test in [opencl/test/unit_test/device/get_device_info_tests.cpp](https://github.com/intel/compute-runtime/pull/374/files#diff-0e66484be04628cb13f248b954c1d5e51154aad97ebfe6a50b39b8b40e65afdb)